# message to store a list of objects.

singleObject[] ObjectList
